app/views/users/edit.html.erb

```diff
@@ -6,42 +6,42 @@
   <div class="col-sm-6">
     <%= form_for @user, as: :user, url: users_path do |form| %>
 
+    <div class="field form-group">
+      <%= form.label :email %>
+      <%= form.text_field :email, class: "form-control", disabled: !current_user.casa_admin? %>
+    </div>
+
+    <div class="field form-group">
+      <%= form.label :display_name %>
+      <%= form.text_field :display_name, class: "form-control" %>
+    </div>
+
+    <%= form.hidden_field :casa_org_id, value: current_user.casa_org_id %>
+
+    <div class="actions">
+      <%= form.submit "Update Profile", class: "btn btn-primary" %>
+      <%= link_to "Back", root_path, class: "btn btn-primary" %>
+      <button class="btn btn-warning accordion" id="accordionExample" type="button" data-toggle="collapse" data-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
+        Change Password
+      </button>
+    </div>
+    <% end %>
+    <br>
+    <div id="collapseOne" class="collapse" aria-labelledby="headingOne" data-parent="#accordionExample">
+      <%= form_for(@user, as: :user, :url => { :action => "update_password" } ) do |f| %>
```

Comment:
minor and non-blocking, but should be `url: { action: `

Reply:
🤦🏻‍♂️ copy&paste strikes again 😂 I'll make the change, thank you.

Reply:
oh, actually that was just formatting changes. I'll make the update.